cvat/settings/base.py

```diff
@@ -234,7 +234,7 @@ def generate_secret_key():
 IAM_ADMIN_ROLE = 'admin'
 # Index in the list below corresponds to the priority (0 has highest priority)
 IAM_ROLES = [IAM_ADMIN_ROLE, 'business', 'user', 'worker']
-IAM_OPA_HOST = 'http://opa:8181'
+IAM_OPA_HOST = os.getenv('IAM_OPA_HOST', 'http://opa:8181')
 IAM_OPA_DATA_URL = f'{IAM_OPA_HOST}/v1/data'
```

Comment:
@jackylamhk Thank you for the contribution!
I would suggest splitting this into 2 env variables. This approach is more flexible:
`CVAT_IAM_OPA_HOST` and `CVAT_IAM_OPA_PORT` and use them to generate `IAM_OPA_DATA_URL`.
```suggestion
CVAT_IAM_OPA_SCHEME = os.getenv('CVAT_IAM_OPA_SCHEME', 'http'),
CVAT_IAM_OPA_HOST = os.getenv('CVAT_IAM_OPA_HOST', 'opa')
CVAT_IAM_OPA_PORT = os.getenv('CVAT_IAM_OPA_HOST', '8181')
IAM_OPA_DATA_URL = f'{CVAT_IAM_OPA_SCHEME}://{CVAT_IAM_OPA_HOST}:{CVAT_IAM_OPA_PORT}/v1/data'
```

Reply:
Good idea, thanks.